src/lib/badwords.ts

```diff
@@ -534,6 +534,14 @@ export default {
 			reason: "discord nitro scam phrase",
 			regex: false,
 		},
+		{
+			match: "Who is first? :)",
```

Comment:
this is kinda generic. are you sure there isn't a more precise phrase you can check for?

Reply:
I know what you mean but i dont thing anyone would send a message like that